tests/acceptance/pageObjects/FilesPageElement/SharingDialog/collaboratorsDialog.js

```diff
@@ -199,7 +199,7 @@ module.exports = {
         await this.api.elementIdClick(collaboratorAvatarElId)
         await this.waitForElementNotPresent(
           '@collaboratorAccessDetailsDrop',
-          this.api.globals.waitForNegativeConditionTimeout
+          this.api.globals.waitForConditionTimeout
```

Comment:
Indeed, sometimes the element takes more than `300ms` (`waitForNegativeConditionTimeout`) to be not present.
![Screenshot from 2022-10-12 17-08-51](https://user-images.githubusercontent.com/52366632/195332341-2044514c-ed91-4021-a687-cdce32f1f562.png)

LGTM :+1:

Reply:
@amrita-shrestha, can PR title be more meaningful? (related to what your changes are)